### ISSUE TITLE
Buttons have blue text

### ISSUE THREAD

**Issue by kjaymiller** [BlackPythonDevs/blackpythondevs.github.io#283]
![CleanShot 2024-03-18 at 18 14 18@2x](https://github.com/BlackPythonDevs/blackpythondevs.github.io/assets/8632637/153b945f-add4-43a5-907e-b42883e51ac8)

The text color should be white
